05-history.md

```diff
@@ -220,3 +220,43 @@ moving backward and forward in time becomes much easier.
 >     $ git checkout <unique ID of last commit> data_cruncher.py
 >     ~~~
 > 5. Both 2 & 4
+
+
+> ## Understanding Workflow and History {.challenge}
+>
+> What is the output of the following commands?
+>
+> ~~~ {.bash}
+> $ cd planets
+> $ nano venus.txt #input the following text: Venus is beautiful and full of love
```

Comment:
Have we already introduced `echo something > file`?  If not, I think 
it'd be better to stick with editing the file.